daniellamarr/warren-web
# Setup responsive mobile navbar

#### What does this PR do?
- This PR creates a responsive navigation bar
#### Description of Task to be completed?
- Create a responsive navbar
- Fix the scroll behavior
#### How should this be manually tested?
- Pull this branch `add-responsive-mobile-navbar`
- Navigate to the root `index.html` file
- Access the webpage in mobile view
- You should see a menu icon, click on the menu icon to see the menu items
#### Screenshots (if appropriate)
<img width="351" alt="Screen Shot 2019-04-29 at 7 02 02 PM" src="https://user-images.githubusercontent.com/42177767/56916713-85ea6f00-6ab1-11e9-9dda-9d7f2ea0f89d.png">
